tests/lib/cli.js

```diff
@@ -85,44 +79,6 @@ describe("cli", () => {
 			"./shared/runtime-info": RuntimeInfo,
 		});
 
-		/**
-		 * Verify that ESLint class receives correct opts via await cli.execute().
-		 * @param {string} cmd CLI command.
-		 * @param {Object} opts Options hash that should match that received by ESLint class.
-		 * @param {string} configType The config type to work with.
-		 * @returns {void}
-		 */
-		async function verifyESLintOpts(cmd, opts, configType) {
-			const ActiveESLint = configType === "flat" ? ESLint : LegacyESLint;
-
-			// create a fake ESLint class to test with
-			const fakeESLint = sinon.mock().withExactArgs(sinon.match(opts));
-
-			Object.defineProperties(
-				fakeESLint.prototype,
-				Object.getOwnPropertyDescriptors(ActiveESLint.prototype),
-			);
-			sinon.stub(fakeESLint.prototype, "lintFiles").returns([]);
-			sinon
-				.stub(fakeESLint.prototype, "loadFormatter")
-				.returns({ format: sinon.spy() });
-
-			const localCLI = proxyquire("../../lib/cli", {
-				"./eslint": { LegacyESLint: fakeESLint },
-				"./eslint/eslint": {
-					ESLint: fakeESLint,
-					shouldUseFlatConfig: () =>
-						Promise.resolve(configType === "flat"),
-				},
-				"./shared/logging": log,
-			});
-
-			await localCLI.execute(cmd, null, configType === "flat");
-			sinon.verifyAndRestore();
-		}
-
-		// verifyESLintOpts
-
```

Comment:
I still need to update some tests and I will reintroduce this function if necessary.